### PR TITLE
Update Niceoppai.kt

### PR DESCRIPTION
**Pull request by ponekub083**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by alessandrojean** [comment]
Please follow the checklist

> Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions

**Comment by ponekub083** [comment]
- [ t ]

**Comment by ponekub083** [comment]
> Please follow the checklist
> 
> > Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions

updated. thx

**Review by Riztard — CHANGES_REQUESTED** [review]
just remove if not being used

File: multisrc/overrides/madara/niceoppai/src/Niceoppai.kt [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+import org.jsoup.nodes.Element
 import okhttp3.Request
 
 class Niceoppai : Madara("Niceoppai", "https://www.niceoppai.net", "th") {
-    private fun pagePath(page: Int) = if (page > 1) "$page/" else ""
-    override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/manga_list/all/any/most-popular/${pagePath(page)}", headers)
-    override fun latestUpdatesRequest(page: Int): Request = GET("$baseUrl/manga_list/all/any/last-updated/${pagePath(page)}", headers)
+    // private fun pagePath(page: Int) = if (page > 1) "$page/" else ""
+    // override fun popularMangaRequest(page: Int): Request = GET("$baseUrl/manga_list/all/any/most-popular/${pagePath(page)}", headers)
+    // override fun latestUpdatesRequest(page: Int): Request = GET("$baseUrl/manga_list/all/any/last-updated/${pagePath(page)}", headers)
```
Comment (Riztard) [inline multisrc/overrides/madara/niceoppai/src/Niceoppai.kt:12]: remove if unused

File: multisrc/overrides/madara/niceoppai/src/Niceoppai.kt [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
+    override val popularMangaUrlSelector = "div.mng_lts_chp div.nde"
+    private fun parseMangaFromElement(element: Element): SManga {
+        val manga = SManga.create()
+
+        with(element) {
+            select(popularMangaUrlSelector +" div.det a").first()?.let {
+
+                manga.setUrlWithoutDomain(it.attr("abs:href"))
+                manga.title = it.ownText()
+                // manga.url = manga.url.removePrefix("/manga")
```
Comment (Riztard) [inline multisrc/overrides/madara/niceoppai/src/Niceoppai.kt:23]: remove if unused

File: multisrc/overrides/madara/niceoppai/src/Niceoppai.kt [hunk 3]
```diff
@@ -1,11 +1,54 @@
...
+    //     "Name (A-Z)", "Name (Z-A)" , "Last Updated","Oldest Updated", "Most Popular", "Most Popular (Weekly)", "Most Popular (Monthly)","Least Popular","Last Added" "Early Added","Top Rating", "Lowest Rating"
+    // )
+    
+
+    // override val orderByFilterOptionsValues: Array<String> = arrayOf(
+    //     "name-az", "name-za" , "last-updated","oldest-updated", "most-popular","most-popular-weekly","most-popular-monthly","least-popular","last-added","early-added", "top-rating", "lowest-rating"
+    // )
+
+    
+
```
Comment (Riztard) [inline multisrc/overrides/madara/niceoppai/src/Niceoppai.kt:52]: this too, remove if unused

**Review by happywillow0 — COMMENTED** [review]
For your awareness, the build check failed and the following should be changed:

MadaraGenerator.kt:356:67: Lint error > [comma-spacing] Missing spacing after ","
Niceoppai.kt:3:1: Lint error > [import-ordering] Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end
Niceoppai.kt:5:1: Lint error > [no-unused-imports] Unused import
Niceoppai.kt:7:1: Lint error > [no-unused-imports] Unused import
Niceoppai.kt:15:45: Lint error > [op-spacing] Missing spacing after "+"
Niceoppai.kt:19:1: Lint error > [no-blank-line-before-rbrace] Unexpected blank line(s) before "}"
Niceoppai.kt:22:45: Lint error > [op-spacing] Missing spacing after "+"
Niceoppai.kt:25:1: Lint error > [no-blank-line-before-rbrace] Unexpected blank line(s) before "}"
Niceoppai.kt:33:61: Lint error > [no-multi-spaces] Unnecessary space(s)

**Comment by alessandrojean** [comment]
These would be likely pointed out during extension tests and building if Android Studio was used. Please do make use of it.

**Comment by ponekub083** [comment]
> 



> 

i will try

**Comment by ponekub083** [comment]
> These would be likely pointed out during extension tests and building if Android Studio was used. Please do make use of it like the checklist asks.

updated . run the extension on Android Studio

**Comment by stevenyomi** [comment]
Find `MadaraGenerator` in the top right of Android Studio and run it once. It will format your code automatically.

**Pull request closed without merging**